cmd/retrieve.go

```diff
@@ -116,19 +115,23 @@ func writeResultsToDisk(results []*io.ReadCloser, fileCreator FileCreator, direc
 
 	for _, result := range results {
 
-		tmpFileName, err := createTemporaryZipFile(result)
-		if err != nil {
-			return err
-		}
+		if !nozip {
+			tmpFileName, err := createTemporaryZipFile(result)
+			if err != nil {
+				return err
+			}
 
-		// unzip the contents of our temp zip file
-		err = unzip(tmpFileName, targetDir, pathMap, fileCreator, directoryCreator, existingFileReader)
+			// unzip the contents of our temp zip file
+			err = unzip(tmpFileName, targetDir, pathMap, fileCreator, directoryCreator, existingFileReader)
 
-		// schedule cleanup of temp file
-		defer os.Remove(tmpFileName)
+			// schedule cleanup of temp file
+			defer os.Remove(tmpFileName)
 
-		if err != nil {
-			return err
+			if err != nil {
+				return err
+			}
+		} else {
+			fmt.Println("Hi")
```

Comment:
Should this still be here?

Reply:
...nope. Told you my brain was fried. I'm working on implementing it now. :-)